api/client/apitask/api_fetch_task.go

```diff
@@ -0,0 +1,62 @@
+package apitask
```

Comment:
I feel that package should move to a place closer to the component that needs to orchestrate api calls based on dependencies. I'm not sure the api package should have a dependency on the tasker